gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -102,57 +100,10 @@ def _is_list_like(value: Any) -> bool:
 def _check_event(
     event: AlertEvent, condition: AlertCondition, method: AlertMethod
 ):
-    if event == AlertEvent.TASK_RUN_STATUS_CHANGED:
```

Comment:
I am really not sure about changing `_check_event` as I don't understand the changes at the moment.